packages/app/src/components/sidebar-metric/sidebar-barscale.tsx

```diff
@@ -82,7 +85,9 @@ export function SidebarBarScale<T>({
   );
 
   assert(
-    text.barscale_screenreader_text,
+    typeof text === 'object' &&
+      'barscale_screenreader_text' in text &&
+      text.barscale_screenreader_text,
```

Comment:
Wouldn't it be possible to use simple optional chaining like this?

```suggestion
      text?.barscale_screenreader_text
```

Reply:
Maybe, but this is mostly done for type narrowing so that has to keep working/be tested